### PR TITLE
High cap magazines for .35 guns

### PR DESCRIPTION
**Pull request by TheShown911**
## About The Pull Request

Adds high cap magazine support for colt, mk58 and paco.

## Why It's Good For The Game

Highcap magazines are used in very few guns, and with this they could be used for every .35 pistol except the giskard.

## Changelog
:cl:
add: Colt, mk58 and paco can use high cap magazines.
/:cl:

### CODE REVIEW

**Review by genessee596 — APPROVED** [review]
delegate stamp of approval

mmyes good

still needs maintainer approval yo

**Review by mattroks101 — APPROVED** [review]

**Pull request merged**
